app/assets/stylesheets/screens/_papers.scss

```diff
@@ -222,7 +222,7 @@ html.control-bar-sub-nav-active #paper-body .oo-ui-toolbar {
 }
 
 .task-list {
-  padding: 40px 30px 0 30px;
+  padding: 15px 30px 0 30px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `15px 30px 0` instead of `15px 30px 0 30px`